ansible_collections/arista/avd/roles/eos_designs/docs/tables/cloudvision-tags-device-custom.md

```diff
@@ -0,0 +1,15 @@
+=== "Table"
+
+    | Variable | Type | Required | Default | Value Restrictions | Description |
+    | -------- | ---- | -------- | ------- | ------------------ | ----------- |
+    | [<samp>cloudvision_tags_device_custom</samp>](## "cloudvision_tags_device_custom") | List, items: Dictionary |  |  |  | List of user defined tags and their values to be applied to this<br>device on CVP.<br> |
+    | [<samp>&nbsp;&nbsp;- name</samp>](## "cloudvision_tags_device_custom.[].name") | String | Required |  |  | Tag name to be assigned to generated tag.<br> |
+    | [<samp>&nbsp;&nbsp;&nbsp;&nbsp;value</samp>](## "cloudvision_tags_device_custom.[].value") | String | Required |  |  | Value to be assigned to the tag.<br> |
+
+=== "YAML"
+
+    ```yaml
+    cloudvision_tags_device_custom:
```

Comment:
I chose to prefix all the keys with `cloudvsision_tags_` so that it's easier to identify them then if the `_tag` part was at the end. I agree it's not easy to read though.

Reply:
All variables have now been changed to use the prefix `cv_tags_` as discussed in the chat.